api/definitions.yaml

```diff
@@ -162,6 +187,8 @@ CreatedMessageWithoutContent:
       $ref: "#/Timestamp"
     sender_service_id:
       type: string
+    payment_data:
```

Comment:
I would say that payment_data is a (structured) part of the message content, we are not going to store it unless the user enabled the inbox, so `MessageWithoutContent` will not include it.

Reply:
moved under content